client/hotrod-client/pom.xml

```diff
@@ -120,7 +120,13 @@
          <scope>test</scope>
       </dependency>
 
-      <dependency>
+       <dependency>
```

Comment:
duplicated dependency. it already depends on commons (see above, first dependency)

Reply:
I have no idea how this landed there....